src/main/java/mate/controller/car/AddDriverToCarController.java

```diff
@@ -32,6 +34,6 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
         Driver driver = driverService.get(driverId);
         Car car = carService.get(carId);
         carService.addDriverToCar(driver, car);
-        resp.sendRedirect("/cars/drivers/add");
+        resp.sendRedirect("/cars/login");
```

Comment:
do you have servlet for this mapping "/cars/login" ?

Reply:
I hanen't( Fixed!